tensorboard/webapp/metrics/views/card_renderer/scalar_card_test.ts

```diff
@@ -3456,6 +3468,25 @@ describe('scalar card', () => {
       store.overrideSelector(getCardStateMap, {});
     });
 
+    it('does not render expand button when the datatable does not overflow', fakeAsync(() => {
+      store.overrideSelector(
+        selectors.getCurrentRouteRunSelection,
+        new Map([
+          ['run1', true],
+          ['run2', true],
+          ['run3', false],
+          ['run4', false],
+        ])
+      );
+      const fixture = createComponent('card1');
+      fixture.detectChanges();
+      const component = fixture.debugElement.query(
+        By.directive(ScalarCardComponent)
+      );
+      expect(component.componentInstance.dataTableContainer).toBeDefined();
+      expect(fixture.debugElement.query(By.css('.expand-button'))).toBeNull();
```

Comment:
Do you have reasoning for using toBeNull vs toBeFalsy? We should try to start being more consistent on this.

Reply:
I don't have a strong opinion here, `toBeNull` is stricter and I tend to lean on the side of strictness for tests.